src/controller/MainScreenController.java

```diff
@@ -47,4 +49,13 @@ public void onViewSourceReports() {
     public void onViewMap() {
         main.setMapScene();
     }
+
+    @FXML
+    public void onSubmitQualityReport() {
+        User currentUser = main.getActiveUser();
+        PermissionLevel permissionLevel = currentUser.getPermissionLevel();
+        if (permissionLevel.getLevel() == 1 || permissionLevel.getLevel() == 2) {
```

Comment:
These shouldn't be magic numbers. Change it to `PermissionLevel.WORKER` and `PermissionLevel.MANAGER`